server/migrations/20190407044853-create-clap.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+module.exports = {
+  up: function(queryInterface, Sequelize) {
+    return Promise.resolve()
+  },
+
+  down: function(queryInterface) {
+    return Promise.resolve()
+  }
+};
```

Comment:
Hi @samorahno  You can include a new line after the last line of code to avoid this EOL error.

Reply:
Thanks @Chiazokam . Fixed